tests/smoke/ckeditor.js

```diff
@@ -14,77 +14,8 @@ bender.test( {
 		removeAllInstances();
 	},
 
-	test_replaceId: function() {
+	'test it loads with the plugin enabled': function() {
 		CKEDITOR.replace( 'editor1' );
 		assert.isObject( CKEDITOR.instances.editor1, 'editor instance not found' );
-		assert.areSame( 'editor1', CKEDITOR.instances.editor1.name, 'instance name doesn\'t match' );
-		assert.areSame( document.getElementById( 'editor1' ), CKEDITOR.instances.editor1.element.$, 'instance element doesn\'t match' );
 	},
-
```

Comment:
originally i had just copied this entire test suite over from CKEditor as proof I had figured out how testing works.

now it's just a smoke test that the editor can load with the plugin.